homeassistant/components/influxdb.py

```diff
@@ -52,6 +53,7 @@ def setup(hass, config):
     ssl = util.convert(conf.get(CONF_SSL), bool, DEFAULT_SSL)
     verify_ssl = util.convert(conf.get(CONF_VERIFY_SSL), bool,
                               DEFAULT_VERIFY_SSL)
+    blacklist = conf.get('blacklist', [])
```

Comment:
I'm not sure how to do this properly.

If I define a global variable CONF_BLACKLIST = [] it will be a mutable global variable which could lead to problems if it is not a read only variable. Are global variable meant to be read only ?
Otherwise it would mean define CONF_BLACKLIST = None and then have some test like
````
if blacklist is None:
    blacklist = []
````
What do you think is the best alternative ?